app/frontend/src/bulk/components/BulkAquiferDocuments.vue

```diff
@@ -62,18 +62,38 @@
 
           <div v-if="behaviourPicked">
             <b-card id="instructions" class="mb-3" title="Instructions">
-              <div v-if="multiBehaviourPicked">
+              <div>
                 <ol>
-                  <li :class="{active: this.multiActiveStep === 'one'}">Use the file picker below to choose one or more documents.</li>
-                  <li :class="{active: this.multiActiveStep === 'two'}">Search for the aquifer in the dropdown list to the right.</li>
-                  <li :class="{active: this.multiActiveStep === 'three'}">Add more aquifers as needed.</li>
-                  <li :class="{active: this.multiActiveStep === 'four'}">Click “Submit” to upload the {{this.upload_files.length}} documents for {{this.aquiferIds.length}} aquifers</li>
-                </ol>
-              </div>
-              <div v-else>
-                <ol>
-                  <li :class="{active: this.keyedActiveStep === 'one'}">Use the file picker below to choose one or more documents keyed by aquifer ID (e.g. factsheet_0001.pdf, factsheet_0002.pdf).</li>
-                  <li :class="{active: this.keyedActiveStep === 'two'}">Click “Submit” to upload the {{this.upload_files.length}} documents for {{Object.keys(this.aquiferDocuments).length}} aquifers</li>
+                  <li v-if="multiBehaviourPicked" :class="{active: multiActiveStep === 'one'}">Use the file picker below to choose one or more documents.</li>
+                  <li v-else :class="{active: keyedActiveStep === 'one'}">Use the file picker below to choose one or more documents keyed by aquifer ID (e.g. factsheet_0001.pdf, factsheet_0002.pdf).</li>
+                  <li v-if="multiBehaviourPicked" :class="{active: multiActiveStep === 'two'}">Search for the aquifer in the dropdown list to the right.</li>
+                  <li v-if="multiBehaviourPicked" :class="{active: multiActiveStep === 'three'}">Add more aquifers as needed.</li>
+                  <li :class="{active: multiActiveStep === 'four' || keyedActiveStep === 'two'}">
+                    Click “Submit” to upload the
```

Comment:
It's probably not as big of an issue with most things anymore, but I tend to gravitate changing “ to " to avoid encoding nightmares 😂

Reply:
Yea I totally understand. I tested Vue before to see how well it handles Unicode outside of ASCII and it seems like it is no problem. :-)